my-app/src/components/Footer.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import "../styles/homepage.css";
+const Footer = () => {
+  return (
+    <div>
+      <footer class="footer flex">
+        <div class="footer-left">
+          <h4>Quick Links</h4>
+          <ul class="footer-nav-links">
+            <li>
+              <a href="#">Home</a>
+            </li>
+            <li>
+              <a href="#">Products</a>
+            </li>
+            <li>
+              <a href="#">My Cart</a>
```

Comment:
Why using a tag ? 
You can use Link from react-router-dom
By using a tag your dom will reload again.